pyvista/utilities/reader.py

```diff
@@ -2027,6 +2031,24 @@ class MetaImageReader(BaseReader):
     _class_reader = _vtk.vtkMetaImageReader
 
 
+class NIFTIReader(BaseReader):
+    """NIFTI Reader for .nii and .nii.gz files.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> filename = examples.download_brain_atlas_with_sides(load=False)
+    >>> filename.split("/")[-1]  # omit the path
+    'avg152T1_RL_nifti.nii.gz'
+    >>> reader = pyvista.get_reader(filename)
+    >>> mesh = reader.read()
+    >>> mesh.plot()
+    """
```

Comment:
PyVista's convention is to end with a newline.

```suggestion

    """
```